### DISCUSSION TITLE
Is python 3.9 or 3.10 also supported?

### DISCUSSION THREAD

**Discussion by jan-kreischer** [IBM/federated-learning-lib#125] [Q&A]
In the setup instructions (https://github.com/IBM/federated-learning-lib/blob/main/setup.md) only python==3.6 is mentioned to be used. I tried to do the same setup on my Mac when using python 3.9. However, I could not make it work.
Does someone know if there is an explicit restriction to use python 3.6?

**Comment by riverallzero** [comment]
Hi @jan-kreischer, i hope my answer will be helpful😀
According to the setup instructions for the IBM Federated Learning Library, Python 3.6 is the recommended version to use. While it may be possible to use other versions of Python, it is possible that there could be compatibility issues with the library and its dependencies.
**So i recommend using Python 3.6 as specified in the setup instructions!!**
